src/time_series_metadata_store.jl

```diff
@@ -682,32 +682,75 @@ function get_time_series_counts_by_type(store::TimeSeriesMetadataStore)
 end
 
 """
-Return a DataFrame with the number of time series by type for components and supplemental
+Return a DataFrame with the number of static time series for components and supplemental
 attributes.
 """
-function get_time_series_summary_table(store::TimeSeriesMetadataStore)
+function get_static_time_series_summary_table(store::TimeSeriesMetadataStore)
     query = """
         SELECT
             owner_type
             ,owner_category
             ,time_series_type
-            ,time_series_category
             ,initial_timestamp
             ,resolution_ms
             ,count(*) AS count
+            ,length AS time_step_count
         FROM $ASSOCIATIONS_TABLE_NAME
+        WHERE time_series_category = "$(_get_time_series_category(StaticTimeSeries))"
         GROUP BY
             owner_type
             ,owner_category
             ,time_series_type
             ,initial_timestamp
             ,resolution_ms
+            ,length
```

Comment:
I would expect that this term needs to match lines 697 and 713.

Reply:
I will fix this.